modules/appeals_api/config/locales/en.yml

```diff
@@ -5,7 +5,8 @@ en:
         hearing_type_preference_inclusion: "If '/data/attributes/boardReviewOption' 'direct_review' or 'evidence_submission' is selected, '/data/attributes/hearingTypePreference' must not be selected"
         hearing_type_preference_missing: "If '/data/attributes/boardReviewOption' 'hearing' is selected, '/data/attributes/hearingTypePreference' must also be present"
         mismatched_ssns: "Request header 'X-VA-SSN' does not match the associated appeal's SSN"
-        outside_legal_window: "This evidence submission is outside the 90 day legal window for submission."
+        nod_outside_legal_window: "This evidence submission is outside the 90 day legal window for submission."
+        sc_outside_submission_window: "This submission is outside of the 7-day window for evidence submission."
```

Comment:
An alternate would be to pass in a translation argument for the timeframe like we did for the `appeals_api.errors.missing_uuid` translation. This way you 1) wouldn't need to define extra methods on the models, and 2) wouldn't need separate translation keys. Everything could still live at the controller level.

e.g:
```
I18n.t('appeals.outside_legal_window', timeframe: "90 day legal")
I18n.t('appeals.outside_legal_window', timeframe: "7-day")
```

Reply:
I didn't go that route because one actually refers to a legal window, and one refers to our PII expunge schedule.